plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -89,7 +89,7 @@ def __init__(self) -> None:
             MimeType(
                 name="application/x-curaproject+xml",
                 comment="Cura Project File",
-                suffixes=["curaproject.3mf"]
+                suffixes=["3mf"]
```

Comment:
I don't think this will work. It either overwrites the MimeType `3MFReader` registers or the other way around.

The project file detection is done with `checkIsValidProjectFile()` in `CuraApplication.py`, and regardless wether the given file is a "3mf" or not, it always performs the check. I think it's better removing this mime type registration to avoid that overwriting problem.